lib/decorators/component-list.js

```diff
@@ -178,7 +178,9 @@ function containsChildComponents(uri, path, el) {
       !child.classList.contains('kiln-inactive-placeholder') &&
       !child.classList.contains('mini-selector') &&
       !child.classList.contains('mini-selector-inactive')) {
-      log.error(`Template for ${getComponentName(uri)} → ${path} contains non-component elements! All direct children of a component list must be components.`, { action: 'containsChildComponents', nonChildEl: child });
+      log.error(`Template for ${getComponentName(uri)} → ${path} contains non-component elements! 
+        All direct children of a component list must be components.`, { action: 'containsChildComponents', nonChildEl: child });
+      log.debug('Ensure component has .hbs template and a single first child with and a data-uri property');
```

Comment:
I like the extra detail but maybe something more like "Ensure each child component has a handlebars template and a single root element with a data-uri property"?